### PR TITLE
Dashboards: Add tips about FROM template variables

### PR DESCRIPTION
**Pull request by rhetoric101**
This is for NR-265156

### CODE REVIEW

File: src/content/docs/query-your-data/explore-query-data/dashboards/dashboard-template-variables.mdx [hunk 1]
```diff
@@ -637,6 +645,48 @@ This results in the values for the last 30 minutes. When you set the time picker
...
+<Collapser
+    id="from-data-types"
+    title="Use variables to select your `FROM` data type"
+>
+
+  Sometimes you have queries that apply the same rules to different types of data. Here are examples of two widgets that are identical except for the data type:
+
+  
+  * WIDGET 1: 
+      ```
```
Comment (chmartinez) [inline src/content/docs/query-your-data/explore-query-data/dashboards/dashboard-template-variables.mdx:657]: @rhetoric101 
the only suggestion I would make is to use the same language syntax as other examples in this page.
Thanks!

Comment (rhetoric101) [reply src/content/docs/query-your-data/explore-query-data/dashboards/dashboard-template-variables.mdx]: I switched these to single-line snippets.

**Review by homelessbirds — APPROVED** [review]
This is very clear! I was able to follow along in the UI. Did this come in as a hero request?

**Comment by rhetoric101** [comment]
@homelessbirds this was for NR-265156.

**Pull request merged**